x-pack/plugins/security_solution/public/hosts/pages/navigation/host_risk_tab_body.tsx

```diff
@@ -86,7 +87,9 @@ const HostRiskTabBodyComponent: React.FC<
     [setOverTimeToggleStatus]
   );
 
-  const rules = data && data.length > 0 ? data[data.length - 1].risk_stats.rule_risks : [];
+  const lastHosttRiskItem: HostRiskScore | null =
+    data && data.length > 0 ? data[data.length - 1] : null;
+  const rules = lastHosttRiskItem ? lastHosttRiskItem.host.risk.rule_risks : [];
```

Comment:
this variable name has a typo, but... is it really needed? 🤷

Reply:
Thanks! Took me a while to understand that you were talking about `lastHosttRiskItem` and not `rules` 😖. I extracted `lastHostRiskItem` so I don't have to repeat it twice. But I could use `lodash/fp last` and make it more readable.